pyproject.toml

```diff
@@ -32,6 +32,7 @@ dependencies = [
   "typing_extensions",
   "scipy",
   "portpicker",
+  "wrapt",
```

Comment:
[good](https://github.com/GrahamDumpleton/wrapt/blob/develop/blog/README.md) package. already pre-installed in colab (it's used by [tensorflow](https://github.com/tensorflow/tensorflow/blob/c7d809a92a3a0db26f9a1ecc0349af430b6c934b/tensorflow/python/data/util/structure.py#L20))

Reply:
Darn - Is there no typed version? Or does python's type system just not robust enough to type? Might be good to look at alternatives just for the sake of eliminating coverage "holes" and for the sake of this repo staying relatively modern.